dotcom-rendering/src/components/StickyBottomBanner.importable.tsx

```diff
@@ -255,19 +252,27 @@ export const StickyBottomBanner = ({
 		setAsyncArticleCounts(getArticleCounts(pageId, tags, contentType));
 	}, [contentType, tags, pageId]);
 
-	useOnce(() => {
-		if (!countryCode) return;
+	useEffect(() => {
+		// Wait for the following dependencies before checking for CMP, Braze + RRCP messages
+		if (
+			isUndefined(countryCode) ||
+			isUndefined(isSignedIn) ||
+			isUndefined(brazeMessages) ||
+			isUndefined(asyncArticleCounts) ||
+			isUndefined(signInGateWillShow) ||
+			isSignedIn === 'Pending'
+		) {
+			return;
+		}
 		const CMP = buildCmpBannerConfig();
 
 		const readerRevenue = buildReaderRevenueBannerConfig(
 			remoteBannerSwitch,
 		)({
-			isSignedIn,
+			isSignedIn: isSignedIn ?? false,
```

Comment:
ESLint righlty marks this as incorrect:

> Unnecessary conditional, expected left-hand side of `??` operator to be possibly null or undefined.

Did you mean `isSignedIn: isSignedIn === true`?

Reply:
good spot! This can just be `isSignedIn`, as we've checked for `Pending` above